Library/Homebrew/ast_constants.rb

```diff
@@ -32,9 +32,6 @@
   [{ name: :on_linux, type: :block_call }],
   [{ name: :on_arm, type: :block_call }],
   [{ name: :on_intel, type: :block_call }],
-  *MacOSVersions::SYMBOLS.keys.map do |os_name|
-    [{ name: :"on_#{os_name}", type: :block_call }]
-  end,
```

Comment:
This is being temporarily removed so that it can be re-ordered in the future (but we need to allow the re-ordering to occur in homebrew/core first). These lines will be re-added in https://github.com/Homebrew/brew/pull/13539